extra/migrations/20160413000000_migrate_puppet_settings.rb

```diff
@@ -80,8 +70,6 @@ def migrate_puppet_configuration(to_migrate)
       end
     end
 
-    migrated[:puppet][:puppet_version] = puppet_version
-
     migrated
   end
 
```

Comment:
While I understand that you need to get rid of "require puppet" in this file, generally you should not change passed migrations. I'd expect a new migration that would remove/rename all "legacy" options.

Reply:
The reason I'm changing this, is that it requires the Puppet gem but we know it's not going to be present. I could mock something up but thought it was easier to modify the migration.